sdk/schemaregistry/azure-schemaregistry-avroencoder/migration_guide.md

```diff
@@ -0,0 +1,194 @@
+# Guide for migrating to azure-schemaregistry-avroencoder v1.0.x from azure-schemaregistry-avroserializer v1.0.0b4
```

Comment:
is the .x to allow for bumps w/o specifying

Reply:
yep!